.github/workflows/build-and-test.yml

```diff
@@ -319,3 +332,19 @@ jobs:
           pip install -r "tests/requirements-cu${TEST_CUDA_MAJOR}.txt"
           pytest -rxXs tests/
           popd
+
+  doc:
+    # The build stage could fail but we want the CI to keep moving.
+    if: ${{ github.repository_owner == 'nvidia' && always() }}
+    # Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+    permissions:
+      id-token: write
+      pages: write
+      contents: read
```

Comment:
I realize this is wrong, fixing it in https://github.com/NVIDIA/cuda-python/pull/338.